roles/captiveportal/tasks/main.yml

```diff
@@ -104,11 +112,11 @@
     state: absent
   when: not captiveportal_enabled
 
-#- name: Restart dnsmasq
-#  systemd:
-#    name: dnsmasq
```

Comment:
This 5-line comment goes with the dozen-lines-below-it.

So prob should be preserved (or the dozen-lines-below should themselves be modified?)

Reply:
removed comment

Reply:
If we're no longer supporting Ubuntu 16.04 (which is likely!) the following 2 stanzas should be combined into one, using `state: restarted`

(Especially if you're removing the explanatory comments.)

Reply:
kind of out side the scope but that could be done later sure.

Reply:
Indeed.  Comments attached to code allow for maintainability, so we should preserve the comments or make the change.

Reply:
Save that for the install/enable split that you are going to preform in the future.